sound/soc/sof/ops.h

```diff
@@ -555,6 +555,14 @@ snd_sof_set_mach_params(struct snd_soc_acpi_mach *mach,
 		sof_ops(sdev)->set_mach_params(mach, sdev);
 }
 
+static inline bool
+snd_sof_is_dai_supported(struct snd_sof_dev *sdev, u32 dai_type, bool dspless_mode)
+{
+	if (sof_ops(sdev) && sof_ops(sdev)->is_dai_supported)
+		return sof_ops(sdev)->is_dai_supported(sdev, dai_type, dspless_mode);
+	return false;
```

Comment:
I prefer to have a blank line between two returns like other ops function in this file.

Reply:
done